### DISCUSSION TITLE
Error failed to run config for nvim-jdtls

### DISCUSSION THREAD

**Discussion by JonathanPinilla** [LazyVim/LazyVim#4520] [Q&A] [answered]
Hi everything was working fine till I updated Lazy, then after opening a java file it shows that error message failed to run config for nvim-jdtls

`lazyvim/plugins/extras/lang/java.lua:79` module `lspcofig.server_configurations.jdtls` not found 

![Screenshot 2024-10-01 at 11 32 58](https://github.com/user-attachments/assets/45f79f84-7757-433b-95d6-5d6eae497307)

**Comment by dpetka2001** [comment, marked as answer]
This will be fixed once #4518 gets merged to fix this error.

**Reply by JonathanPinilla** [reply]
thank you
